v4-client-js/src/clients/modules/composer.ts

```diff
@@ -445,6 +450,51 @@ export class Composer {
     };
   }
 
+  // ------------ x/vault ------------
+  public composeMsgDepositToMegavault(
+    address: string,
+    subaccountNumber: number,
+    quoteQuantums: Uint8Array,
+  ): EncodeObject {
+    const subaccountId: SubaccountId = {
```

Comment:
[not related to this PR] wait, subaccountId is an object with an address and a number? feels like a weird name for this type.